### PR TITLE
In-App review in kDrive

### PR DESCRIPTION
**Pull request by tevincent**
[inapp_review_kdrive.webm](https://github.com/Infomaniak/android-kDrive/assets/149579879/bacdc78f-3761-4c6e-9e86-956b344975a1)

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt [hunk 1]
```diff
@@ -147,6 +149,15 @@ class MainActivity : BaseActivity() {
     private val inAppUpdateManager by lazy { InAppUpdateManager(this, BuildConfig.APPLICATION_ID, BuildConfig.VERSION_CODE) }
     private var inAppUpdateSnackbar: Snackbar? = null
 
+    private val inAppReviewManager by lazy {
+        InAppReviewManager(
```
Comment (FabianDevel) [inline app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt:153]: put arguments name

File: app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt [hunk 2]
```diff
@@ -271,6 +283,18 @@ class MainActivity : BaseActivity() {
     private fun canDisplayInAppSnackbar() = inAppUpdateSnackbar?.isShown != true
     //endregion
 
+    //region In-App Review
+
```
Comment (FabianDevel) [inline app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt:287]:
```suggestion
```

File: app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt [hunk 3]
```diff
@@ -271,6 +283,18 @@ class MainActivity : BaseActivity() {
...
+    private fun initAppReviewManager() {
+        inAppReviewManager.init(
+            onDialogShown = { trackInAppReview("presentAlert") },
+            onUserWantToReview = { trackInAppReview("like") },
+            onUserWantToGiveFeedback = { trackInAppReview("dislike") },
+        )
+    }
+
```
Comment (FabianDevel) [inline app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt:295]:
```suggestion
```

**Review by FabianDevel — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt [hunk 4]
```diff
@@ -147,6 +149,15 @@ class MainActivity : BaseActivity() {
     private val inAppUpdateManager by lazy { InAppUpdateManager(this, BuildConfig.APPLICATION_ID, BuildConfig.VERSION_CODE) }
     private var inAppUpdateSnackbar: Snackbar? = null
 
+    private val inAppReviewManager by lazy {
+        InAppReviewManager(
+            activity = this,
+            reviewDialogTheme = R.style.DialogStyle,
+            reviewDialogTitleResId = R.string.reviewAlertTitle,
+            feedbackUrlResId = R.string.urlUserReportAndroid
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt:157]:
```suggestion
            feedbackUrlResId = R.string.urlUserReportAndroid,
```

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — APPROVED** [review]

**Review by FabianDevel — APPROVED** [review]

**Pull request merged**